packages/ui/src/app/pages/Council/Council.tsx

```diff
@@ -32,16 +43,39 @@ export const Council = () => {
 
   const { council, isLoading } = useElectedCouncil()
   const { idlePeriodRemaining, budget, reward } = useCouncilStatistics()
+  const periodInformation = useCouncilPeriodInformation()
+
+  const endsAt = useMemo(
+    () =>
+      periodInformation && council
+        ? {
+            number: periodInformation.periodEnds[3],
+            timestamp: new Date(
+              new Date(council.electedAt.timestamp).getTime() +
+                (periodInformation.periodEnds[3] - council.electedAt.number) * MILLISECONDS_PER_BLOCK
+            ).toISOString(),
+          }
+        : { number: 0, timestamp: new Date().toString() },
+    [periodInformation, council]
+  )
+
   const { activities } = useCouncilActivities()
 
   const [order, setOrder] = useState<CouncilOrder>({ key: 'member' })
   const { councilors, isLoading: isLoadingCouncilors } = useCouncilorWithDetails(council)
   const sortedCouncilors = useMemo(() => councilors.sort(sortBy(order)), [councilors])
-  const header = <PageHeaderWithHint title="Council" hintType="council" tabs={<CouncilTabs />} />
+  const header = (
+    <PageHeaderWithHint
+      title={`Council ${council ? '#' + Number(council.electionCycleId) : ''}`}
```

Comment:
It's better to use:
```suggestion
      title={`Council ${council ? '#' + parseInt(council.id, 36) : ''}`}
```
We didn't have a failed election until now, so `council.electionCycleId` was so far equal to council term number, but this will change now, because of failed election cycle 45 (so the council for term 45 will be elected in election cycle 46 or even later)